R/readNWISunit.R

```diff
@@ -493,14 +514,18 @@ readNWISgwl <- function(siteNumbers,
     service = "gwlevels",
     format = "rdb"
   )
-
+  
   data <- importRDB1(
     obs_url = url,
     asDateTime = TRUE,
     convertType = convertType,
     tz = tz
   )
-
+  
+  if(!is.na(parameterCd)){
+    data <- data[data$parameter_cd %in% parameterCd, ]
+  }
```

Comment:
Sorry if I missed this in our meeting: does this mean that the service sometimes pulls parameters that weren't specified in the `parameterCd` input?